portfolio/src/main/java/com/google/sps/servlets/FollowedTagsServlet.java

```diff
@@ -52,7 +52,8 @@
 @WebServlet("/follow-tags")
 public class FollowedTagsServlet extends HttpServlet {
   List<String> followedPosts= new ArrayList<>();
```

Comment:
followedPosts doesn't seem to be used anywhere. For something like this, you should check it in with the code that uses it. If it makes sense to set up some code beforehand, leave comments and a TODO.

I'm also curious about the usage of this. It usually not a good idea to have dynamic member variables in Servlets because servlets process many requests and servlets are not guaranteed to persist across mutliple requests by the same user. Can you please explain this?